src/app/(tabs)/home/styles.ts

```diff
@@ -22,6 +22,7 @@ const styles = StyleSheet.create({
   scrollView: {
```

Comment:
The `subheading` style should have `marginBottom: 8` (because of the padding 8 below)